apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx

```diff
@@ -123,6 +126,8 @@ export const SwipeUpDown = forwardRef<SwipeUpDownRef, SwipeUpDownProps>(
         enablePanDownToClose={false}
         enableHandlePanningGesture={!disabled}
         enableContentPanningGesture={!disabled}
+        activeOffsetX={[-SCREEN_WIDTH, SCREEN_WIDTH]}
+        activeOffsetY={[-10, 10]}
```

Comment:
is the y offset no-op? not sure I follow why we need it

Reply:
But it was not intuitive for me as well so I tried to do it with only `activeOffsetX` and was wondering why all bottom sheet doesn't work 🫠

Reply:
sounds good, looks like other people do similar numbers too: https://github.com/search?q=activeOffsetX%3D+&type=code